rpcserver.go

```diff
@@ -2434,7 +2559,8 @@ func SendToAddress(w *Wallet, chainSvr *chain.Client, icmd btcjson.Cmd) (interfa
 		cmd.Address: btcutil.Amount(cmd.Amount),
 	}
 
-	return sendPairs(w, chainSvr, cmd, pairs, 1)
+	// sendtoaddress always debits the default account, this matches bitcoind
```

Comment:
I meant to provide rationale for passing `waddrmgr.DefaultAccountNum` in `sendPairs`. I've updated to say "spends from the default account" .